cli/cmd/cluster_create.go

```diff
@@ -79,6 +81,16 @@ func (r *runners) createCluster(_ *cobra.Command, args []string) error {
 		Tags:                   tags,
 		DryRun:                 r.args.createClusterDryRun,
 	}
+	if r.args.createClusterMinNodeCount >= 0 {
+		opts.MinNodeCount = &r.args.createClusterMinNodeCount
+	} else if r.args.createClusterMinNodeCount != -1 {
```

Comment:
Can this be a string input and the value parsed in code here?  Then just check if it's empty sting to use default values.